### PR TITLE
nsc-events-android-1-49-ui-create-forgot-password-page

### PR DESCRIPTION
**Pull request by taylorpapke**
resolves: https://github.com/SeattleColleges/nsc-events-android/issues/49

Screenshot 2023-10-07 at 2 23 21 PM

### CODE REVIEW

**Comment by tinpham5614** [comment]
Everything is looking good to me. One thing just want to comment is:
On Belinda's App, I used Scaffold to create a Top App Bar with a Back Arrow. I'm not sure about NCS. 
I have the tutorial here for reference: [https://www.youtube.com/watch?v=EqCvUETekjk](https://www.youtube.com/watch?v=EqCvUETekjk) (4:04 for Top App Bar)

File: app/src/androidTest/java/com/example/nsc_events/ForgotPasswordTest.kt [hunk 1]
```diff
@@ -0,0 +1,52 @@
+package com.example.nsc_events
```
Comment (bcko) [inline app/src/androidTest/java/com/example/nsc_events/ForgotPasswordTest.kt:1]: It's great to see that you are adding tests.

**Comment by bcko** [comment]
> resolves: #49
> 
> Screenshot 2023-10-07 at 2 23 21 PM

The image illustrates that the back button is not positioned in the top-left corner. 

Here is a documentation that you can reference
- scaffold: https://developer.android.com/jetpack/compose/components/scaffold
- app bars: https://developer.android.com/jetpack/compose/components/app-bars

**Review by Seiyaroo — APPROVED** [review]
Dark mode check is great. Like BC said, the location of the back arrow might be a bit low, so once that is fixed, I believe we can approve this for merge! 👍

File: app/src/androidTest/java/com/example/nsc_events/ForgotPasswordTest.kt (hunk 1 above)
Comment (theGaryLarson) [reply app/src/androidTest/java/com/example/nsc_events/ForgotPasswordTest.kt:1]: Yes, thanks @taylorpapke was looking it over now. I like how it automates entries into the text input. Pretty Neat!

**Review by theGaryLarson — APPROVED** [review]
Looks great! Will need to add backend functionality to handle forgot password route in the User Controller & Service @Exochos 

Mentioned Jeremy Ward here (Exochos) so he can slot this for another item to work on. As he is in the process of building out the User controller & service on Nest.js.

**Review by ogarmela — APPROVED** [review]

**Review by ogarmela — DISMISSED** [review]

**Comment by taylorpapke** [comment]
Screenshot 2023-10-10 at 1 21 46 PM

**Comment by taylorpapke** [comment]
@Seiyaroo Might be a good idea to add TopAppBar for back button to the requirements for #44 and add the tutorials above?

**Comment by Seiyaroo** [comment]
> @Seiyaroo Might be a good idea to add TopAppBar for back button to the requirements for #44 and add the tutorials above?

Done! @taylorpapke definitely makes sense

**Review by theGaryLarson — APPROVED** [review]
Let's include the top Nav bar. Just as Tin had done in Belinda's closet. It will make our work easier in the long run. We can create a separate issue for this and fix the back button then.

**Review by Annelisebx — APPROVED** [review]
The UI looks correct. I was able to create an almost identical UI for Belinda's closet 'forgot password' page. [PR 65](https://github.com/SeattleColleges/belindas-closet-android/pull/65) resolving [Issue 43](https://github.com/SeattleColleges/belindas-closet-android/issues/43)

**Pull request merged**